app/helpers/me/notices_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Me::NoticesHelper
+  def dissmiss_all_notices_button
+    link_to_remote_with_confirm(:dismiss_all_notices.t,
```

Comment:
I try to avoid `link_to_remote...` in new code and prefer `link_to, ... , remote: true`. This will make the migration away from prototype easier at some point. I will mark the remote functions as deprecated.